docs/evaluation/examples.rst

```diff
@@ -517,3 +525,18 @@ earlier doesn't group the model to match the data)::
 
 Note that this example has not tried to subtract the background
 or fit it!
+
+Reference/API
```

Comment:
I don't think you meant to add this here. We already have these linked to in the data section: https://sherpa--1182.org.readthedocs.build/en/1182/data/index.html#reference-api

I've been trying to only add the Reference/API sections to the "foo/index.rst" files and not "sub" files like "foo/bar.rst".

Reply:
Again, I think that's a sign of a bad rebase. Fixing!